templates/run_template.sh

```diff
@@ -6,9 +6,12 @@ OPENSHIFT_PASS="admin"
 DOCKERHUB_USER="CHANGEME"
 DOCKERHUB_PASS="CHANGEME"
 DOCKERHUB_ORG="ansibleplaybookbundle"
+REGISTRY_TYPE="dockerhub"
+REGISTRY_URL="docker.io"
+DEV_BROKER="true"
 
```

Comment:
I think this depends on https://github.com/openshift/ansible-service-broker/pull/166